apps/molgenis-components/src/components/tables/RowButtonAdd.vue

```diff
@@ -18,10 +18,14 @@
 
 <script>
 import RowButton from "./RowButton.vue";
+import { defineAsyncComponent } from "vue";
 
 export default {
   name: "RowButtonAdd",
-  components: { RowButton },
+  components: {
+    RowButton,
+    EditModal: defineAsyncComponent(() => import("../forms/EditModal.vue")),
```

Comment:
Max and I also tried to add EditModal here last week, but we couldn't figure out how to do it. good to know defineAsyncComponent is the way to go!